src/underwear.css

```diff
@@ -258,10 +258,14 @@ textarea {
   padding-right: 20px;
 }
 
+input[type='number'] {
+  padding-right: 0;
+  padding-left: 12px;
+}
+
 select:not([multiple]) {
-  background-color: var(--editable-background-color);
-  background: url('data:image/svg+xml;utf8, <svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" version="1.1" width="20" height="20" x="120px" y="0px" viewBox="0 0 100 100" style="enable-background:new 0 0 100 100;" xml:space="preserve"><g><polygon points="79.3,42.1 76.4,39.3 51.3,65 25.5,39.9 22.7,42.7 51.3,70.7  "></polygon></g></svg>')
-    center right no-repeat;
+  background: url('data:image/svg+xml;utf8, <svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" version="1.1" width="20" height="20" x="120px" y="0px" viewBox="0 0 100 100" style="enable-background:new 0 0 100 100;" xml:space="preserve"><g><polygon points="79.3,42.1 76.4,39.3 51.3,65 25.5,39.9 22.7,42.7 51.3,70.7  "></polygon></g></svg>') center right no-repeat;
```

Comment:
Can you change this to `background-image` to avoid overwriting background color. It might confuse us later. Then you'll need to remove `center right no-repeat` and add new rule: `background-repeat: no-repeat`.

Reply:
Done, good catch!